frontend/src/metabase/query_builder/components/DatasetEditor/DatasetFieldMetadataSidebar/DatasetFieldMetadataSidebar.styled.jsx

```diff
@@ -24,7 +24,7 @@ export const AnimatableContent = styled.div`
 const CONTENT_PADDING = "24px";
 
 const FormContainer = styled.div`
-  ${Radio.BaseItem} {
+  ${RadioGroup} {
```

Comment:
Without exposing RadioGroup it would be:

const FormRadio = styled(Radio)`
  color: ${color("text-dark")};
`;